apps/services/auth/admin-api/src/app/v2/secrets/me-client-secrets.controller.ts

```diff
@@ -60,29 +64,39 @@ export class MeClientSecretsController {
     response: { status: 201, type: ClientSecretDto },
   })
   @Audit<ClientSecretDto>({
-    resources: (secret) => secret.id,
+    resources: (secret) => secret.secretId,
   })
   create(
     @CurrentUser() user: User,
     @Param('tenantId') tenantId: string,
     @Param('clientId') clientId: string,
-    @Body() input: CreateClientSecretDto,
```

Comment:
The `Body` in import becomes unused an can be removed.

Reply:
Fixed.